### PR TITLE
Ensure that labels for mesos tasks launched via MesosCommandExecutor are collected

### PR DESCRIPTION
**Pull request by sashankreddya**
This is an add on to issue 1965

### CODE REVIEW

**Comment by dashpole** [comment]
let me know when you would like this merged (or if you are waiting for review from anyone else).

**Comment by sashankreddya** [comment]
sure David. Will let you know by EOD tomorrow.

File: container/mesos/factory.go [hunk 1]
```diff
@@ -107,16 +107,11 @@ func (self *mesosFactory) CanHandleAndAccept(name string) (handle bool, accept b
 	// Check if the container is known to mesos and it is active.
 	id := ContainerNameToMesosId(name)
 
-	c, err := self.client.ContainerInfo(id)
+	_, err = self.client.ContainerInfo(id)
```
Comment (dashpole) [inline container/mesos/factory.go]: do we still need to make this call?

Comment (sashankreddya) [reply container/mesos/factory.go]: Yes as part of factory, we need to see if the container info object  can be created with the given id.

**Pull request merged**